examples/BPZ_lite_with_custom_SEDs.ipynb

```diff
@@ -58,7 +58,7 @@
    "id": "a0d9bdc4-6022-4668-b003-686489e13ccd",
    "metadata": {},
    "source": [
-    "This should have successfully copied the files to the proper SED directory. Now, we can proceed in the same manner that we did in the `BPZ_lite_demo.ipynb` notebook:"
+    "This should have successfully copied the files to the proper SED directory. Now, we can proceed in the same manner that we did in the `BPZliteEstimator_demo.ipynb` notebook:"
```

Comment:
query replace changed this name where it shouldn't have

Reply:
Good catch, thanks!